robot.html

```diff
@@ -911,6 +911,18 @@ <h1><a href="index.html" style="text-decoration:none;">Robotic Manipulation</a><
       </ol>
     </exercise>    
 
+  </h1></section>
+
+    <exercise><h1>Direct Joint Teleop in Drake</h1>
+
+      <p> For this exercise you will implement a method for controlling the joints of a robot in Drake. You will work exclusively in <script>document.write(notebook_link('direct_joint_control', deepnote['exercises/robot'], link_text='this notebook'))</script>, and should use the <a href="https://deepnote.com/workspace/Manipulation-ac8201a1-470a-4c77-afd0-2cc45bc229ff/project/1-Introduction-6e32ce10-e406-4026-abfb-00be78c478cb/%2Fintro.ipynb">example notebook in chapter 1 as a reference</a>. You will be asked to complete the following steps: </p>
```

Comment:
we should use `notebook_link` to make the intro chapter link, too.  I'll make the change myself.